### DISCUSSION TITLE
Is it possible to involve skywalking java agent through maven

### DISCUSSION THREAD

**Discussion by legendtkl** [apache/skywalking#8470] [Q&A] [answered]
I want to involve skywalking java agent into my project, but I don't want to uphold the jar artifact in git repo.

Is it possible to add maven dependency for skywalking java agent? and then I can retieve the java agent jar from my project distribution.

Thanks.

**Comment by wu-sheng** [comment, marked as answer]
Strange question. The agent has nothing related to maven, which is a compiling and packaging tool. 
You could download the file through the Maven plugin if you want. But this is on your own, it is out of SkyWalking's scope.

**Reply by legendtkl** [reply]
Thanks.
